DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -271,7 +271,9 @@
    <version ClassVersion="11" checksum="1239840459"/>
   </class>
 
-  <class name="pat::PackedCandidate" ClassVersion="19">
+  <class name="pat::PackedCandidate" ClassVersion="23">
+    <version ClassVersion="23" checksum="663492232"/>
+    <version ClassVersion="21" checksum="1075665714"/>
```

Comment:
I investigated this, and the problems already exist.  7_5_X and 7_6_X are inconsistent with 7_4_X, in that version 23 has a different checksum in 7_4_X the later releases. This should be fixed, and I will fix it. However, this PR does not make the problem worse, so, since you already did a +1 for this, I can wait till after this PR is merged to fix the underlying problem. Since 7_4_0 is the mature release, it will not be modified. Rather, 7_5_X and 7_6_X will be modified to be consistent with 7_4_X.